packages/shared/src/components/post/PostNavigation.tsx

```diff
@@ -54,49 +38,58 @@ export function PostNavigation({
   };
 
   return (
-    <div
-      className={classNames('flex flex-row gap-2 items-center', className)}
-      role="navigation"
-    >
-      {onPreviousPost && (
-        <SimpleTooltip content="Previous">
-          <Button
-            className="-rotate-90 btn-secondary"
-            icon={<ArrowIcon />}
-            onClick={onPreviousPost}
-          />
-        </SimpleTooltip>
-      )}
-      {onNextPost && (
-        <SimpleTooltip content="Next">
-          <Button
-            className="rotate-90 btn-secondary"
-            icon={<ArrowIcon />}
-            onClick={onNextPost}
+    <>
```

Comment:
I think we can remove the Fragment here.

Reply:
Same comment here, it includes two elements on a root level, thus needs to be wrapped to be a valid ReactElement.